Python/Product/Uwp.Interpreter/InstalledPythonUwpInterpreter.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Python Tools for Visual Studio
+// Copyright(c) Microsoft Corporation
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the License); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY
+// IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+//
+// See the Apache Version 2.0 License for specific language governing
+// permissions and limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+
+namespace Microsoft.PythonTools.Uwp.Interpreter {
+    internal class InstalledPythonUwpInterpreter {
+        static public string GetDirectory(Version ver) {
+            string result;
+            if (_installedSdks.Value.TryGetValue(ver, out result)) {
+                return result;
+            }
+            return null;
+        }
+
+        static public IEnumerable<KeyValuePair<Version, string>> GetInterpreters() {
+            return _installedSdks.Value;
+        }
+
+        static private Lazy<Dictionary<Version, String>> _installedSdks = new Lazy<Dictionary<Version, string>>(DiscoverPythonUwpSdks, LazyThreadSafetyMode.ExecutionAndPublication);
+
+        static private Dictionary<Version, String> DiscoverPythonUwpSdks() {
+            var userSdkInstallDir = Environment.ExpandEnvironmentVariables(@"%LOCALAPPDATA%\Microsoft\VisualStudio\%VISUALSTUDIOVERSION%Exp\Extensions\Microsoft\Python UWP");
+            var sdkInstallDir = Environment.ExpandEnvironmentVariables(@"%ProgramFiles(x86)%\Microsoft SDKs\Windows\v10.0\ExtensionSDKs\Python UWP");
```

Comment:
I didn't make this change.  I merely refactor this out of the PythonUwpInterpreterFactoryProvider class for re-use.  I'll fix it in a later pull request.